kpi/tests/api/test_api_environment.py

```diff
@@ -33,41 +35,44 @@ def setUp(self):
             'community_url': constance.config.COMMUNITY_URL,
             'frontend_min_retry_time': constance.config.FRONTEND_MIN_RETRY_TIME,
             'frontend_max_retry_time': constance.config.FRONTEND_MAX_RETRY_TIME,
-            'project_metadata_fields': lambda x: \
```

Comment:
Sorry about the quantity of formatting changes. The smallest unit of code that I could invoke python-black on was the dictionary.

Reply:
Who am I to complain about receiving few lines of new formatting changes in a review? 😇